scripts/install_third_party.sh

```diff
@@ -42,6 +42,18 @@ install_node_module through2 2.0.0
 install_node_module uglify-js 3.3.11
 install_node_module yargs 3.29.0
 
+install_node_module jasmine-core 2.5.2
+install_node_module karma 1.5.0
+install_node_module karma-jasmine 1.1.0
+install_node_module karma-jasmine-jquery 0.1.1
+install_node_module karma-json-fixtures-preprocessor 0.0.6
+install_node_module karma-coverage 1.1.1
+install_node_module karma-ng-html2js-preprocessor 1.0.0
+install_node_module karma-chrome-launcher 2.0.0
+install_node_module protractor 5.3.1
+install_node_module protractor-screenshot-reporter 0.0.5
+install_node_module jasmine-spec-reporter 3.2.0
+
```

Comment:
These packages are only needed for running Karma frontend tests.

I recommend putting these in a separate shell script like [this](https://github.com/oppia/oppia/pull/5374/commits/c2cecf7f4c61e2375d23dbe7248d2a3b6d2c5ac0) so that these lines are only ran by `run_frontend_tests.sh`

Reply:
Thanks! Good call and I just did that.

But don't we need them in the e2e tests as well?

Reply:
Ah, just checked. The separate script is called from maybeInstallDependencies which seems to show up in both run_frontend_tests.sh and run_e2e_tests.sh.

Reply:
Done -- PTAL.

Reply:
And somehow that is not causing the other e2e tests to suffer the #5333 problem on Travis. Truly a mystery.